src/components/YourCompany/NameTranslations/ListNameTranslation.vue

```diff
@@ -20,9 +20,12 @@
         <v-chip v-if="translation.action === ActionTypes.ADDED"
           x-small label color="#1669BB" text-color="white">ADDED</v-chip>
         <v-chip v-if="translation.action === ActionTypes.EDITED"
-          x-small label color="#1669BB" text-color="white">CORRECTED</v-chip>
+          x-small label color="#1669BB" text-color="white">
+          <span v-if="isCorrectionView()">CORRECTED</span>
+          <span v-else>CHANGED</span>
+        </v-chip>
         <v-chip v-if="translation.action === ActionTypes.REMOVED"
-          x-small label color="#E0E0E0" text-color="grey darken-1">REMOVED</v-chip>
+          x-small label color="#E0E0E0" text-color="grey darken-4">REMOVED</v-chip>
```

Comment:
Please use a named Vuetify color here (and replace any others you find).

Ref: https://vuetifyjs.com/en/styles/colors/

Specifically: https://github.com/vuetifyjs/vuetify/blob/9bec589dcd6ba4630e1ce5473753165cf238b622/packages/vuetify/src/styles/settings/_colors.scss#L377

Reply:
Done